src/tfvc/tfvc.ts

```diff
@@ -29,16 +32,42 @@ export class Tfvc {
             const settings = new TfvcSettings();
             this._tfvcPath = settings.Location;
             if (!this._tfvcPath) {
-                //TODO localize the message once this code stops changing so much
                 throw new TfvcError({
-                    message: "The path to the TFVC command line was not found in the user settings. Please set this value.",
+                    message: Strings.TfvcLocationMissingError,
                     tfvcErrorCode: TfvcErrorCodes.TfvcNotFound
                 });
             }
         }
-        //TODO check to make sure that the file exists in that location
-        //TODO check the version of TFVC command line
 
+        // check to make sure that the file exists in that location
+        const stats: any = fs.lstatSync(this._tfvcPath);
+        if (!stats || !stats.isFile()) {
+                throw new TfvcError({
```

Comment:
We should chat about our understanding of how TfvcError works.  It *is* a way to create objects that we can throw but we can't *catch(){}*  a particular type... which reduces its effectiveness.  Nothing to do for this comment, just FYI.